json-logic/schema.ts

```diff
@@ -0,0 +1,146 @@
+// LogicExpression refers to the JSONLogic standard.
+import { LogicExpression } from "./logic";
+
+/*
+* This module defines a schema for questionaire and result calulation logic.
+*/
+
+/**
+ * Type of a question, essentially defines the result type.
+ */
+export enum QuestionType {
+	Select = "Select",
+	Multiselect = "Multiselect",
+	Number = "Number",
+	Boolean = "Boolean",
+	Date = "Date",
+	Text = "Text"
+}
+
+/**
+ * Option for multi-select questions.
+ */
+export interface IOption {
+	/**
+	 * Human-Readable answer, can be localized.
+	 */
+	text: string,
+	/**
+	 * Value used for evaluating logic expressions.
+	 */
+	value: string;
+}
+
+/**
+ * Represents a question.
+ */
+export interface IQuestion {
+	/**
+	 * Unique id for referring this question in logic expressions.
+	 */
+	id: string
+	/**
+	 * Type of the question.
+	 */
+	type: QuestionType,
+	/**
+	 * Human-readable question text, can be localized.
+	 */
+	text: string,
```

Comment:
Further information, like examples or detailed description. 
```suggestion
	text: string,
	/**
	 * Details for the Question/Item
	  */
	  definition: string,
```

Reply:
An extra field for detailed information, or better field documentation?

Reply:
for detailed information like: 
"Do you have a fever?" 
--> `definition`: "fever is a body temperature above 37°C for more than 3 hours." 
(Definition is the name of the field from FHIR